src/constants/labels.json

```diff
@@ -17,7 +17,7 @@
     "errorLoadCustomDiscipline": "Fehler: Bereich konnte nicht geladen werden. Ungültiger Schlüssel:",
     "deleteModal": {
       "confirmationQuestion": "Soll der Bereich wirklich gelöscht werden?",
-      "cancel": "Abbrechen",
+      "cancel": "Schließen",
```

Comment:
now we have 3 times the "cancel": "Schließen", please only keep it once and move it to "general"

Reply:
removed the three entries and created one in general